en/contributing/build.md

```diff
@@ -1,28 +1,71 @@
-# Building
+# Building DroneCore from Source
 
-## Build the Library
+To build all of DroneCore from source you will first need to build the C++ library, and then any wrappers. The instructions below show how.
 
-Make sure the submodules are updated first.
+## Build the C++ Library
 
+### Linux and Mac OS X
+
+First install the dependencies
+```bash
+sudo apt-get update -y
+sudo apt-get install cmake build-essential colordiff astyle git libcurl4-openssl-dev -y
 ```
+
+> **Note** If the build reports a missing dependency, confirm that the set above matches the requirements in the [Dockerfile](https://github.com/dronecore/DroneCore/blob/master/Dockerfile).
+
+Clone the DroneCore repository (or your fork) and update the submodules:
+
+```bash
+git clone https://github.com/dronecore/DroneCore.git
+cd DroneCore
 git submodule update --init --recursive
 ```
 
-### Desktop
-
-Build for the desktop to test and debug:
+You can then build the library by calling:
 
 ```
 make default
 ```
 
-### Android
+### Windows
+
+To build DroneCode on Windows you'll need to install Visual Studio Community 2015 with Update 3 (this is an older version, but is still available through the free Microsoft Dev Essentials Program).
```

Comment:
I added this, because VC2015 isn't the "standard build". Any reason not to use a later version that is more readily available?